index.html

```diff
@@ -228,36 +221,67 @@ <h2 class="about-head">About me</h2>
+      <section class="contact">
+        <div class="form-detail">
           <p>
-            <button class="button" type="submit">Get in touch</button>
-           </p>
-  </form>
-  <footer class="social-footer-social">
-  <div class="social-media-icon1">
-    <a href="#"  class="social-link">
-      <img class="social-icon" src="./icons/github.png" alt="GitHub"/>
-    </a>
-    <a href="#"  class="social-link">
-      <img class="social-icon" src="./icons/Linkedin.png" alt="LinkedIn"/>
-    </a>
-    <a href="#"  class="social-link">
-      <img class="social-icon" src="./icons/angel.png" alt="Angel"/>
-    </a>
-    <a href="#"  class="social-link">
-      <img class="social-icon" src="./icons/twitter.png" alt="Twitter"/>
-    </a>
-    <a href="#"  class="social-link">
-    <img class="social-icon" src="./icons/medium.png" alt="Medium"/>
-    </a>
-  </div>
-  </footer>
-</section>
-</main>
-</body>
+            I'm always interested in hearing about new projects, so if you'd
+            like to chat please get in touch.
+          </p>
+        </div>
+        <div class="contact-form">
+          <form
+            action="https://formspree.io/f/xlezlkyw"
+            method="post"
+            class="contact-us"
+          >
+            <input
+              type="text"
+              placeholder="Full Name"
+              maxlength="30"
+              required
+            />
+            <input type="email" placeholder="Email address" required />
+            <textarea
+              name="msgt1"
+              id="t1"
+              cols="30"
+              rows="10"
+              maxlength="500"
+              placeholder="Enter text here"
+              required
+            ></textarea>
+            <p>
+              <button class="button" type="submit">Get in touch</button>
+            </p>
+          </form>
+        </div>
+      </section>
+      <footer class="social-footer-social">
+        <div class="social-media-icon1">
+          <a href="#" class="social-link">
+            <img class="social-icon" src="./icons/github.png" alt="GitHub" />
+          </a>
+          <a href="#" class="social-link">
+            <img
+              class="social-icon"
+              src="./icons/Linkedin.png"
+              alt="LinkedIn"
+            />
+          </a>
+          <a href="#" class="social-link">
+            <img class="social-icon" src="./icons/angel.png" alt="Angel" />
+          </a>
+          <a href="#" class="social-link">
+            <img class="social-icon" src="./icons/twitter.png" alt="Twitter" />
+          </a>
+          <a href="#" class="social-link">
+            <img class="social-icon" src="./icons/medium.png" alt="Medium" />
+          </a>
+        </div>
+        <div class="footer-line-section">
+          <img src="assets/png/footer-line-shape.png" alt="Footer Line Shape" />
+        </div>
+      </footer>
```

Comment:
- Kindly try to stick as much as possible to the Figma design, which includes:
   - [ ] Adding the horizontal line above the social media to match the Figma design
   - [ ] Adjusting the color of the social media icons to match the Figma design.

| Your design | Figma design |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/24703171/160218870-e4ca0f7e-d3a6-4380-8ee2-2d126eb4fae4.png) | ![image](https://user-images.githubusercontent.com/24703171/160218883-2de87c99-238e-4840-9aae-7ab49e0a0e0f.png) |